styles.css

```diff
@@ -0,0 +1,380 @@
+
+main {
+  width: 1300px;
+  margin: auto;
+  z-index: -1;
+  background-color: #fff;
+}
+
+body {
+  background-color: #f2f2f2;
+}
+
+h4 {
+  font-weight: bold;
+  font-size: 22px;
+}
+
+h5 {
+  font-weight: bold;
+  font-size: 18px;
+}
+
+.intro {
+  font-size: 30px;
+  font-weight: bold;
+}
+
+.navig {
+  background-color: #ff1a1a;
+  width: 1300px;
+  margin: auto;
+  height: auto;
+}
+
+.navig ul li i {
+  font-size: 1.8rem;
+  margin-top: -2px;
+}
+
+.row-after-nav {
+  width: 1300px;
+  margin: auto;
+  background-color: #fff;
+  margin-bottom: 0;
+}
+
+.row-after-nav ul {
+  padding-left: 0;
+  position: relative;
+  bottom: 1rem;
+  margin-left: 0;
+  width: 96%;
+  padding-top: 1rem;
+  border-top-right-radius: 90px;
+  background-color: #fff;
+  z-index: 0;
+}
+
+footer ul li {
+  display: inline;
+}
+
+.row-after-nav ul li {
+  display: inline;
+  margin: 0;
+}
+
+.row-after-nav ul li a {
+  font-size: 15px;
+  font-weight: 900;
+  color: #000;
+  text-decoration: none;
+  padding-right: 0.5rem;
+  padding-left: 0.5rem;
+}
+
+#searchfield {
+  height: 2rem;
+  position: relative;
+  top: 0;
+  width: 6rem;
+  border: none;
+}
+
+#search-icons {
+  position: relative;
+  left: -47px;
+  bottom: -6px;
+}
+
+#rain-logo {
+  height: 1.5rem;
+  width: 1.5rem;
+}
+
+#img-sm {
+  height: 1.5rem;
+  width: 8rem;
+  margin-top: -8px;
+}
+
+.sticky {
+  position: fixed;
+  top: 10;
+  height: 3rem;
+  width: 100%;
+  overflow: hidden;
+  z-index: 1;
+  padding-top: 2rem;
+}
+
+.list1 {
+  margin-top: -8px;
+}
+
+.display-none {
+  visibility: hidden;
+}
+
+.display-block {
+  height: 1.5rem;
+  width: 8rem;
+}
+
+.first-part img {
+  width: 100%;
+  height: auto;
+  image-rendering: crisp-edges;
+}
+
+.sus {
+  padding-left: 0.5rem;
+  padding-right: 0.5rem;
+  padding-top: 0.1rem;
+  padding-bottom: 0.1rem;
+  margin-bottom: 1rem;
+}
+
```

Comment:
We can define margin and padding in one line. For example, `margin: top right bottom left`. We should follow the DRY principle [reference](https://www.w3schools.com/cssref/pr_margin.asp)
Please try to match font-size and font-weight if not font-family of the original site as it is a clone project.